ui/widgets/nuxeo-date.js

```diff
@@ -70,6 +72,10 @@ import './nuxeo-tooltip.js';
         tooltipFormat: String,
       };
     }
+
+    _validateTootip(datetime, format, tooltipFormat) {
+      return this.formatDate(datetime, format) === this.formatDateTime(datetime, tooltipFormat);
```

Comment:
This actually raises an interesting question, more from a testing perspective, but also from a code 🙂 
I'm not sure which approach would be better. 🤔 

1. To check the formats (not only the properties, but also the global ones coming from `Nuxeo.UI.config`) **without calling the methods**.

2. Keeping as it is, but with the "overhead" of **calling the same functions** with the same arguments multiple times;

On both we will have the question: "what if the behavior implementation changes?"

Reply:
yes I had the same question myself. Because the method format does a lot of the checks of defaults formats already, i thought it would be good to reuse. I understand we need to call it multiple times, but else i would call something very similar on my side I think? not sure which is the best approach either

Reply:
Let's wait for more opinions. 🙂

Reply:
Given that the formatting comes at a cost, even if small, best practice would probably be to just compare the result. Then there question is "what if different formats produce the same result?". In the end I feel it's just easier to explain and document that tooltip is hidden when using different formats.

Reply:
@nelsonsilva not sure I understood - do you think I should keep the current logic or change it to compare the formats directly?